Scripts/static-analyzer/Sources/Analyzer/Utilities/Collection.swift

```diff
@@ -0,0 +1,16 @@
+
+extension Sequence {
```

Comment:
Nitpick, the file name is `Collection` but the extension is on `Sequence`.

Reply:
I used `Collection` in the filename meaning this file includes helpers on collection types(Array, Set, Dictionary, etc.), not just the `Swift.Collection` type. But I see the confusion here 😅